tests/pfcwd_test.py

```diff
@@ -192,6 +192,37 @@ def test_pfcwd_start_actions(self, mock_os):
         assert result.exit_code == 0
         assert result.output == pfcwd_show_start_action_drop_output
 
+    @patch('pfcwd.main.os')
+    def test_pfcwd_pfc_not_enabled(self, mock_os):
+        import pfcwd.main as pfcwd
+        runner = CliRunner()
+        db = Db()
+
+        # get initial config
+        result = runner.invoke(
+            pfcwd.cli.commands["show"].commands["config"],
+            obj=db
+        )
+        print(result.output)
+        assert result.output == pfcwd_show_config_output
+
+        mock_os.geteuid.return_value = 0
+        # remove 'pfc_enabled' entry from Ethernet0 node
+        db.cfgdb.mod_entry("PORT_QOS_MAP", "Ethernet0", None)
```

Comment:
One suggestion, instead of modifying here don't add the "PORT_QOS_MAP" entry for one of the ports.

Reply:
Yes, we can do not add "pfc_enabled" entry for  "PORT_QOS_MAP|Ethernet0". 
But in this case output of other testcases will be also affected, because "pfc_enabled" entry now become mandatory and all testcases are using the same config_db files
For example: after this change Ethernet0 will be continuously skipped during pfcwd configuration so output of existing testcases will be different

Reply:
There is no test case that would be affected by not adding. Either way is fine.